src/components/AddTodoForm/AddTodoForm.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import usersFromServer from './../../api/users';
+import { Todo } from '../../types/Todo';
+import { getUserById } from '../../utils/getUserById';
+import { getNewTodoId } from '../../utils/getNewTodoId';
+
+interface Props {
+  onAdd: (todo: Todo) => void;
+  todos: Todo[];
+}
+
+enum TypeErrors {
+  Default = '',
+  TitleError = 'Please enter a title',
+  UserIdError = 'Please choose a user',
+}
+
+export const AddTodoForm: React.FC<Props> = ({ onAdd, todos }) => {
+  const [title, setTitle] = useState('');
+  const [titleError, setTitleError] = useState(TypeErrors.Default);
+
+  const [selectedUserId, setSelectedUserId] = useState(0);
```

Comment:
[CODE STYLE] - Using '0' to represent no user selected can be misleading. It's better to use `null` or `undefined` for uninitialized or empty states. This would also align with the checks you're doing in `handleSubmit` for `!selectedUserId` which would fail if `selectedUserId` is `0`.